english/coffee/Run/run.md

```diff
@@ -0,0 +1 @@
+Research has shown that coffee before a workout improves circulation, lessens pain, improves memory for exercises and routines, and preserves and fuels muscles.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.